app/workers/question_worker.rb

```diff
@@ -10,13 +10,15 @@ class QuestionWorker
   def perform(user_id, question_id)
     user = User.find(user_id)
     question = Question.find(question_id)
+    webpush_app = Rpush::App.find_by(name: "webpush")
 
     user.followers.each do |f|
       next if f.inbox_locked?
       next if f.banned?
       next if MuteRule.where(user: f).any? { |rule| rule.applies_to? question }
 
-      Inbox.create(user_id: f.id, question_id: question_id, new: true)
+      inbox = Inbox.create(user_id: f.id, question_id:, new: true)
+      f.push_notification(webpush_app, inbox)
```

Comment:
might be better to do this instead so that it doesn't break if the rpush-app is not there yet

```suggestion
      f.push_notification(webpush_app, inbox) if webpush_app
```